src/Layers/xrRender/xrRender_console.cpp

```diff
@@ -61,6 +61,10 @@ const xr_token qsun_quality_token[] = {{"st_opt_low", 0}, {"st_opt_medium", 1},
 #endif // !USE_DX9
     {nullptr, 0}};
 
+u32 ps_r_water_reflection = 3;
+const xr_token qwater_reflection_quality_token[] = {{"st_opt_off", 0}, {"st_opt_low", 1}, {"st_opt_medium", 2},
+    {"st_opt_high", 3}, {"st_opt_ultra", 4}, {nullptr, 0}};
```

Comment:
```suggestion
const xr_token qwater_reflection_quality_token[] =
{
    { "st_opt_off", 0 },
    { "st_opt_low", 1 },
    { "st_opt_medium", 2 },
    { "st_opt_high", 3 },
    { "st_opt_ultra", 4 },
    { nullptr, -1 }
};
```

Reply:
Are you sure? It violates the current code style in the file and will look alien.

Reply:
Yes, I'm sure.